SVCPLAN-4023 Update puppet-profile_monitoring to v0.3.0

bsper2 opened the pull request.

billglick reviewed (commented):
A couple of questions:
- The new tag number iterates oddly. Are you sure that is correct?
- Was this tested on a few different types of nodes?

bsper2 commented:
I bumped it up from `v0.2.1` to `v0.3.1` when it should have gone to `v0.3.0`. Deleted the old tag and added the correct one.

It has been tested on control-test-rhel9a and control-test-rhel8a

billglick reviewed (approved):
Looks good.
